src/settings.ts

```diff
@@ -69,7 +69,7 @@ export interface IDebuggingSettings {
 
 export interface IDeveloperSettings {
     featureFlags?: string[];
-    powerShellExePath?: string;
+    // This setting is no longer used but is here to assist in cleaning up the users settings.
```

Comment:
What setting does this comment apply to?

Reply:
oops - removed that.